go.mod

```diff
@@ -3,112 +3,88 @@ module github.com/openshift/splunk-forwarder-operator
 go 1.17
 
 require (
-	cloud.google.com/go v0.47.0 // indirect
-	github.com/Azure/go-autorest/autorest v0.11.18 // indirect
-	github.com/go-logr/logr v0.1.0
-	github.com/go-openapi/spec v0.19.4
-	github.com/gophercloud/gophercloud v0.5.0 // indirect
-	github.com/onsi/ginkgo v1.16.1 // indirect
-	github.com/onsi/gomega v1.11.0 // indirect
-	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/spf13/pflag v1.0.5
-	go.uber.org/multierr v1.2.0 // indirect
+	github.com/openshift/operator-custom-metrics v0.5.0
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.58.0
+	github.com/prometheus-operator/prometheus-operator/pkg/client v0.58.0
+	k8s.io/apimachinery v0.24.3
```

Comment:
I think we wanted to remain on `k8s.io/* v0.23.5` and `controller-runtime v0.11.2` for now per [SDE-1972](https://issues.redhat.com/browse/SDE-1972)

Reply:
I see that goal as more of a "let's bring everything up to this as the bare minimum", as there will inevitably be drift. (cc @rendhalver - your thoughts?) I can always lock in these versions in the `replace`, though.

Reply:
Got it, I'm also in favor of this sentiment (and not a fan of having to `replace` unless we absolutely must haha) - thanks for explaining!